packages/dd-trace/src/latency_stats.js

```diff
@@ -0,0 +1,156 @@
+const os = require('os')
+const pkg = require('../../../package.json')
+const { decodePathwayContext } = require('../../datadog-plugin-kafkajs/src/hash')
+
+const { LogCollapsingLowestDenseDDSketch } = require('@datadog/sketches-js')
+
+const { LatencyStatsExporter } = require('./exporters/latency-stats')
+
+class AggStats {
+  constructor (aggKey) {
+    this.hash = aggKey.hash
+    this.parentHash = aggKey.parentHash
+    this.edgeTags = aggKey.edgeTags
+    this.edgeLatency = new LogCollapsingLowestDenseDDSketch(0.00775)
```

Comment:
What is the significance of the `0.00775` value? It might be better to define it as a constant as a form of documentation:

```js
// replace name with whatever the reason is for the value
const HIGH_ACCURACY_DISTRIBUTION = 0.00775
new LogCollapsingLowestDenseDDSketch(HIGH_ACCURACY_DISTRIBUTION)
```

Reply:
This number was taken directly from how span stats distribution was set up, Roch said unless otherwise specified in the RFC, it should be good to use